model/task/db.go

```diff
@@ -1282,3 +1285,22 @@ func AbortTasksForVersion(versionId string, taskIds []string, caller string) err
 	)
 	return err
 }
+
+func AddHostCreateDetails(taskId, hostId string, hostCreateError error) error {
+	if hostCreateError == nil {
```

Comment:
Do you need to log if there's no error?

Reply:
It seems like we shouldn't be able to get this far with no error, so in case it does happen I thought it could be helpful to have some logging built in so we can investigate; I'm not super tied to it though